aptos-node/src/logger.rs

```diff
@@ -77,5 +77,17 @@ fn log_config_and_build_information(node_config: &NodeConfig) {
     log_feature_info!("failpoints", "assert-private-keys-not-cloneable");
 
     // Log the node config
-    info!(config = node_config, "Loaded AptosNode config");
+    let mut config = node_config;
```

Comment:
why do we want both of these... I might suggest a minor rewrite here...

```
if let Some(u) = ... {
  let mut url = url:...
  parsed_url.set_password(""); // or just unset it
  let mut masked = node_config.clone();
  // set url 
  info();
} else {
  info()
}

Reply:
Thanks. Your suggestion looks clean.
I was wondering is there any particular reason that we use 
`info!(config = node_config, "Loaded AptosNode config")`, 
not 
`info!("Loaded AptosNode config {:?}", &node_config);`
since the first one use `serde#serialize` , so we need this multiple line codes.
If we can use the second one, just one line is enough.